src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -321,51 +310,37 @@ public URL getBuildPlanWebUrl(ProgrammingExerciseParticipation participation) {
         return BAMBOO_SERVER_URL;
     }
 
-    /**
-     * Clones an existing Bamboo plan.
-     *
-     * @param templateProject The Bamboo project in which the plan is contained.
-     * @param templatePlan    The plan's name.
-     * @param toProject       The Bamboo project in which the new plan should be contained.
-     * @param name            The name to give the cloned plan.
-     * @return The name of the new build plan
-     * @throws BambooException if a communication issue occurs or the plan already exists.
-     */
-    public String clonePlan(String templateProject, String templatePlan, String toProject, String name) throws BambooException {
-
-        String toPlan = toProject + "-" + name;
+    @Override
+    public String clonePlan(String templatePlan, String planKey, String planName) throws BambooException {
```

Comment:
Can you add the documentation for this method?

Reply:
So there already is documentation for all overridden methods in the interface of the service. I.m.o. we don't really need to add more JavaDoc if the original documentation is enough. 
Especially since we would only duplicate the doc and might get out of sync in later iterations. This might result in developers reading through different kinds of documentation depending on what class they are looking at. What do you think about this @lovaiible ?

Reply:
Yes, that's ok :)